terminus/geometry/arc.py

```diff
@@ -195,7 +242,7 @@ def heading_at_offset(self, offset):
         if offset > self.length():
             raise ValueError("Offset ({0}) is greater than segment length ({1})".format(offset, self.length()))
         angular_offset = math.copysign(offset, self._angular_length) * 180 / (math.pi * self._radius)
-        return self._theta + angular_offset
+        return (self._theta + angular_offset) % 360
```

Comment:
Sorry, but I don't get what you mean. As this returns a heading, I think that in 2D or 3D is reasonable to expect it between 0 and 360. I get the angular length maybe greater than 360, but not the heading.